scripts/legislation/africa/sierra_leone_english.sh

```diff
@@ -0,0 +1,527 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^Interprelation./,$p' | \
+    sed -n '/^PART I - PRELIMINARY/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function remove_all_text_after_last_article {
+  sed -z 's/2 \.\.\..*$//' 
+}
+
+function amend_errors_in_articles {
+  sed -E 's/- /: /g' | \
+    sed -E 's/of([a-z]+)/of \1/g' | \
+    sed -E 's/\[•\] //g' | \
+    sed -E 's/ \(1\)/ \[1\]/g' | \
```

Comment:
Shouldn't the `replace_parentheses_around_article_delimiters_with_square_brackets` function handle this? I believe the problem is the caret (`^`) at the beginning of the regular expression in the `replace_parentheses_around_article_delimiters_with_square_brackets` function.

Reply:
Yes, I believe you are correct. However, due to the number of edits required in this legislation, I would prefer to leave the code as is instead of making this edit now which will affect the following code.

Reply:
Yes, that's fine.